routes/export.rb

```diff
@@ -0,0 +1,32 @@
+require 'models/user'
+require 'models/resource'
+require 'models/permission'
+require 'models/event'
+require 'csv'
+require 'date'
+
+get '/export/?' do
+    user_events = Event.includes(:event_type).joins(:event_signups).
+		where(:event_signups => {:user_id => @user.id}).
+		where('end_time >= ?', Time.now)
+    trainer_events = Event.
+    where(:events => {:trainer_id => @user.id}).
```

Comment:
Very minor but could move the first where right after Event. and then indent the second where so that the code style matches the user_events above it.

trainer_events = Event.where(:events => {:trainer_id => @user.id}).
        where('end_time >= ?', Time.now)